src/Account.java

```diff
@@ -42,4 +55,26 @@ public int compareTo(Account otherAccount) {
         return Double.compare(this.balance, otherAccount.balance);
```

Comment:
I thought we compare the accounts by calling the getAccountType method that we know each child class implements and comparing them as string? Why are you comparing by account balance? We compare them only when sorting them by account type, not balance.